extensions/transmission/src/listTorrents.tsx

```diff
@@ -120,10 +121,13 @@ export default function TorrentList() {
 
   return (
     <List
-      isShowingDetail={isShowingDetail}
+      isShowingDetail={expanded != null}
       isLoading={!didLoad || loadingStatusFilter}
       searchBarPlaceholder="Filter torrents by name..."
       onSearchTextChange={setSearch}
+      onSelectionChange={(value) => {
+        setTimeout(() => setExpanded((expanded) => (expanded != null ? Number(value) : null)), 100);
```

Comment:
This is a terrible hack, but I can't find another solution to make this work with the current Raycast API.

If panel is open and you press Cmd Num, the primary action is called instead of just navigating to it like it happens when you move with the up/down arrows.
So if the primary action performs a naive “toggle” based on the visibility of the detail panel, it will close it.
If instead you want to make it smarter by checking which element is currently selected, you need to use `onSelectionChange` but you need a timeout because that’s called before the primary action is triggered.